app/src/play/java/com/duckduckgo/referral/PlayStoreAppReferrerStateListener.kt

```diff
@@ -178,7 +178,7 @@ class PlayStoreAppReferrerStateListener @Inject constructor(
     }
 
     private fun getMatchingServices(serviceIntent: Intent): List<ResolveInfo> =
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+        if (Build.VERSION.SDK_INT >= 33) {
```

Comment:
I decided to fix this one because the :app module is the only one that has flavours and, surprise! lint-baselines can't be generated for multiple flavours out of the box.
And it was quicker to fix it than creating ruby code to try and have multiple `lint-baselines` per flavour.